packages/atlas/src/hooks/clipboard/hooks.ts

```diff
@@ -0,0 +1,12 @@
+import { useSnackbar } from '@/providers/snackbars'
```

Comment:
It's in `hooks/` now but it doesn't match how we files in that directory, please take a look at other hooks